flink-libraries/flink-table/pom.xml

```diff
@@ -40,6 +40,12 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-streaming-scala_2.10</artifactId>
 			<version>${project.version}</version>
+			<exclusions>
```

Comment:
I noticed that this change causes the build to fail because `flink-hbase` and `flink-connector-kafka-0.10` depend on `flink-table` to pull required dependencies. The missing dependencies need to be added to their pom files to fix the build.